lib/driver/mongodb.js

```diff
@@ -0,0 +1,330 @@
+var util = require('util');
+var moment = require('moment');
+var MongoClient = require('mongodb').MongoClient;
+var Server = require('mongodb').Server;
+var Base = require('./base');
+var type = require('../data_type');
+var log = require('../log');
+
+var connectionString;
+
```

Comment:
you need to add global.migrationTable = 'migrations'; to specify the migrationTable name within the test.